cli/borges/command.go

```diff
@@ -148,9 +78,12 @@ func (c *metricsOpts) maybeStartMetrics() {
 	if c.Metrics {
 		addr := fmt.Sprintf("0.0.0.0:%d", c.MetricsPort)
 		go func() {
-			logrus.Debug("Started metrics service at", "address", addr)
+			logger := log.New(log.Fields{"address": addr})
```

Comment:
I would keep on the top an instance of `logger := log.New(log.Fields{"address": addr})`
and reuse it below calling `logger.Debug`, etc. If I need a new field then I would use `With`, e.g.:
`logger.With(log.Fields{"error": err}).Warningf("profiler failed to listen and serve")` (or in this case `logger.Errorf(err, "profiler failed to listen and serve")`)

Reply:
Now I've understood. I've changed New to With in those two log lines. Thanks.